.pre-commit-config.yaml

```diff
@@ -9,14 +9,14 @@ repos:
         types: [python]
         entry: scripts/run-in-env.sh ruff check --fix
         require_serial: true
-        stages: [commit, push, manual]
+        stages: [pre-commit, pre-push, manual]
```

Comment:
this change is unrelated to the PR and under normal circumstances shouldnt be needed - can you tell me the motivation to change it ?